enclave_wrangler/objects_api.py

```diff
@@ -1024,6 +1045,80 @@ def get_projects(verbose=False) -> List[Dict]:
     return data
 
 
+def cset_obj_field_datatypes(schema=SCHEMA) -> Dict[str, str]:
+    """Get data types for each field in the code_sets table.
+
+    todo: If there's a use case, would be nice to generalize this function:
+     - support any table, not just code_sets
+     - param to return keys for the DB table model, or Enclave object model (if there is corollary obj model for table)
+    """
+    with get_db_connection(schema='') as con:
+        table_data_types: List[Dict[str, str]] = [dict(x) for x in sql_query(con, f"""
+            SELECT column_name,  data_type FROM information_schema.columns 
+            WHERE table_schema = '{schema}' AND table_name = 'code_sets';""")]
+        table_data_types: Dict[str, str] = {x['column_name']: x['data_type'] for x in table_data_types}
+        obj_data_types: Dict[str, str] = convert_row('code_sets', 'OMOPConceptSet', table_data_types)
+    return obj_data_types
+
+
+def cset_objs_set_missing_fields_to_null(cset_objs: List[Dict], schema=SCHEMA) -> List[Dict]:
+    """Will set any add any fields that don't exist on object and set them to NULL.
+
+    todo: could be nice to have a generalized function for other object types"""
+    # noinspection PyTypeChecker doesnt_understand_union_w_list_of_lists
+    obj_data_types: Dict[str, str] = cset_obj_field_datatypes(schema)
+    obj_fields: Set[str] = set(obj_data_types.keys())
+    # TODO: update it so that it sets to None for everything except numeric type, in which case it should set to 0
+    #   - did my modification below to use PG_DATATYPES_BY_GROUP work? run it
+    return [
+        cset | {
+            f: 0 if obj_data_types[f] in PG_DATATYPES_BY_GROUP['numeric'] else None
+            for f in obj_fields.difference(cset.keys())}
+        for cset in cset_objs
+    ]
+
+
+def update_cset_metadata_from_objs(
```

Comment:
## `update_cset_metadata_from_objs()`
Wrapper for:
1. Updates the 'audit' table
2. Kicks off the updates